html/join.html

```diff
@@ -7,57 +7,118 @@
+        </div>
+
+        <div id="session-list-container">
+          <input
+            class="session-filter-box"
+            id="session-filter-box"
+            type="search"
+            placeholder="Search for sessions..."
+          >
+
+          <table id="session-row-container" class="session-row-container">
+            <template id="session-row-template">
+              <tr class="session-row">
+                <td class="row-label">
+                  <input class="session-option" name="sessionOption" type="radio">
+                </td>
+                <td class="session-name"></td>
+                <td class="session-model-name"></td>
+                <td class="session-info"></td>
+              </tr>
+            </template>
+            <thead class="session-header" id="session-header">
+              <tr>
+                <th>Session Name</th>
+                <th>Activity</th>
+                <th># Participants</th>
+              </tr>
+            </thead>
+            <tbody class="session-rows" id="session-rows"></tbody>
+          </table>
+
+        </div>
+        <hr class="session-list-boundary"></hr>
+
+        <form class="user-inputs-container" id="join-form" action="javascript:void(0);" method="POST">
+          <div class="user-inputs-top-row">
+            <div>
+              <label class="form-label">Username
+                <input class="username" id="username" name="username" type="text" required>
+              </label>
+            </div>
+            <div>
+              <label class="form-label">Password
+                <input class="password" id="password" name="password" type="password" disabled>
+              </label>
+            </div>
+          </div>
+          <div class="user-inputs-bottom-row">
+            <div class="flex-column">
+              <label class="form-label">Role
+                <select id="role-select" class="role-select" disabled></select>
+              </label>
+            </div>
+            <input id="join-button" class="join-button" type="submit" value="Join" disabled>
           </div>
         </form>
-      </div>
 
-      <div id="session-list-container" class="flex-column session-list-container">
-        <input id="session-filter-box" type="search" placeholder="Filter...">
-        <div id="session-option-container" class="flex-column"></div>
-        <template id="session-option-template">
-          <label class="session-label">
-            <span class="session-name"></span> | <span class="session-model-name"></span> | <span class="session-info"></span> connected
-            <input class="session-option" name="sessionOption" type="radio">
-          </label>
-        </template>
       </div>
 
+      <div class="right-column">
+        <img
+          id="session-preview-image"
+          class="session-preview-image"
+          src="http://ccl.northwestern.edu/netlogo/models/models/HubNet%20Activities/Disease%20HubNet.png"
```

Comment:
> I think I can just remove this placeholder since the images actually load correctly on this page now?

I'm not sure that's the right answer.  Is there something we could/should put there to let people know that an image will appear there once a session is clicked in the list?